doc/filters/map.rst

```diff
@@ -2,7 +2,8 @@
 =======
 
 .. versionadded:: 1.41
-    The ``map`` filter was added in Twig 1.41 and 2.10.
+
+    The ``map`` filter was added in Twig 1.41.
```

Comment:
the removal of `and 2.10` looks wrong to me. This was done on purpose because 1.x and 2.x were developed in parallel (1.x was not in bugfix-only mode after the 2.0 release). So a feature added in 1.41 is not available in 2.0 even though the version number is higher (and having to switch version of the doc to discover that would not be done by devs which don't know about this specific case).

Reply:
this is not the only affected place in the diff.

Reply:
I agree with @stof, these changes should be reverted

Reply:
To me it is fine to develop in parallel, but in this case the feature needs a versionadded directive in `2.x` and `1.x`, otherwise someone using `2.x` and browsing the `2.x` doc would not know when this method was introduced 🤔 

To fix this problem, I would remove the mention of `1.4.1` in `2.x`:
![CleanShot 2020-11-05 at 11 58 36](https://user-images.githubusercontent.com/995707/98232750-56e27280-1f5e-11eb-93af-fae09753d254.png)

As this was developed in parallel we have two new introduced methods, one per version

Reply:
ok, fair enough

Reply:
it should indeed have a version added in 2.x too mentioning 2.10. But the message in 1.x should keep mentioning both 1.41 and 2.10 IMO.

Reply:
btw, maybe the message in 2.x should also mention both in the details (so that people don't think that this feature is entirely unavailable in 1.x), changing only the main version (or maybe we should support passing multiple version to the directive directly)